tests/driver_ds3231/main.c

```diff
@@ -258,6 +262,70 @@ static int _cmd_test(int argc, char **argv)
         return 1;
     }
 
+    /* clear all existing alarm flag */
+    res = ds3231_clear_alarm_1_flag(&_dev);
+    if (res != 0) {
+        puts("error: unable to clear alarm flag");
+        return 1;
+    }
+
+    /* get time to set up next alarm*/
+    res = ds3231_get_time(&_dev, &time);
+    if (res != 0) {
+        puts("error: unable to read time");
+        return 1;
+    }
+
+    time.tm_sec += TEST_DELAY;
+    mktime(&time);
+
+    /* set alarm */
+    res = ds3231_set_alarm_1(&_dev, &time, DS3231_AL1_TRIG_H_M_S);
+    if (res != 0) {
+        puts("error: unable to program alarm");
+        return 1;
+    }
+
+#ifdef MODULE_DS3231_INT
+
+    /* wait for an alarm with GPIO interrupt */
+    res = ds3231_await_alarm(&_dev);
+    if (res < 0){
+        puts("error: unable to program GPIO interrupt or to clear alarm flag");
+    }
+
+    if (!(res & 0x01)){
```

Comment:
I'd prefer 0x1 to be a define (`DS3231_ALARM_FLAG_1` maybe)

Reply:
Yes, a define is way better. Not sure where I was supposed to put it so I added it at the top of the test script.